webhooks/wh-general.md

```diff
@@ -130,22 +82,15 @@ and you may receive multiple `ServiceOrderUpdated` events in one Webhook message
 
 ### Event discriminator
 
-* `ServiceOrderUpdated` - A [Reservation](../operations/reservations.md#reservation) or other Service Order was updated. `Value` is [Entity updated data](#entity-updated-data).
-* `ResourceUpdated` - A [Resource](../operations/resources.md#resource) (for example a guest room or other space) was updated. `Value` is [Entity updated data](#entity-updated-data).
-* `MessageAdded` - A [Message](../operations/messages.md#message) was added. `Value` is [Entity updated data](#entity-updated-data).
-* `ResourceBlockUpdated` - A [Resource block](../operations/resourceblocks.md#resource-block) was updated. `Value` is [Entity updated data](#entity-updated-data).
+* `ServiceOrderUpdated` - A [Reservation](../operations/reservations.md#reservation) or other Service Order was updated. `Value` is [Entity updated data](#entity-updated-data). Data about entity can be obtained by [Get all reservations](../operations/reservations.md#get-all-reservations).
```

Comment:
I'd transform this into table:
| Discriminator | Description | Value | Fetch operation |

Reply:
I think that's probably a good idea

Reply:
Migrated to table